types/any_type.go

```diff
@@ -0,0 +1,35 @@
+package types
```

Comment:
Should add license identifier `// SPDX-License-Identifier: Apache-2.0` to the file

Reply:
added, Thanks